python/examples/sample-selection.py

```diff
@@ -0,0 +1,74 @@
+import sys
+
+import chemfiles
+
+from rascaline import SoapPowerSpectrum
+from equistore import Labels, TensorMap
+import numpy as np
+
+with chemfiles.Trajectory(sys.argv[1]) as trajectory:
+    frames = [f for f in trajectory]
+
+HYPER_PARAMETERS = {
+    "cutoff": 5.0,
+    "max_radial": 6,
+    "max_angular": 4,
+    "atomic_gaussian_width": 0.3,
+    "center_atom_weight": 1.0,
+    "radial_basis": {
+        "Gto": {},
+    },
+    "cutoff_function": {
+        "ShiftedCosine": {"width": 0.5},
+    },
+}
+
+calculator = SoapPowerSpectrum(**HYPER_PARAMETERS)
+
+descriptor = calculator.compute(frames)
+
+# The names of the samples that will be chosen during training
+# coincide with the samples of the blocks obtained after calculating the
+# properties. Accordingly, the names of the samples can be obtained with the
+# following commands (usually 'structure' (structure number) and 'center'
+# (atom number in the structure).
```

Comment:
I don't understand these sentences. What are you trying to say?

Reply:
> I don't understand these sentences. What are you trying to say?

That the names of samples to be used for selected_samples are the same as those for blocks in the resulting descriptor

Reply:
right, but this is not the case (the names in the selection need to be a subset of the samples names from the calculator, not necessarily the same ones). How about something like

```
The selections for sample can be a set of `Labels`, in which case the names of the labels must be 
a subset of the names of the samples produced by the calculator. You can see the default set of names with:
```